caiman/motion_correction.py

```diff
@@ -1642,7 +1642,7 @@ def register_translation(src_image, target_image, upsample_factor=1,
                          "and \"fourier\" values for the ``space`` argument."
 
     References:
-    .. [1] Manuel Guizar-Sicairos, Samuel T. Thurman, and James R. Fienup,
+    .. [1] Manual Guizar-Sicairos, Samuel T. Thurman, and James R. Fienup,
```

Comment:
Can you remove this change?

Reply:
sure, will force push rewrite, thanks for catching